.travis.yml

```diff
@@ -0,0 +1,43 @@
+language: php
+
+php:
+  - 5.3
+  - 5.4
+  - 5.5
+  - 5.6
+  - 7.0
+  - hhvm
+
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer
+
+env:
+  - SYMFONY_VERSION=2.8.*
+
+matrix:
+  include:
+    - php: 5.6
+      env: SYMFONY_VERSION=2.3.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.7.*
+    - php: 7.0
+      env: SYMFONY_VERSION=2.8.*@dev
+    - php: 7.0
+      env: SYMFONY_VERSION=^3.0
```

Comment:
This ensures to be tested against the latest stable version of 3.x and ending up with the LTS if released ^^